akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -1013,3 +1013,20 @@ Related config properties: `akka.cluster.use-dispatcher = akka.cluster.cluster-d
 Corresponding default values: `akka.cluster.use-dispatcher =`.
 
 @@@
+
+### Configuration Compatibility Check
+
+Creating a cluster is about deploying two or more nodes and make then behave as if they were one single application. Therefore it's extremely important that all nodes in a cluster are configured with the same settings. 
```

Comment:
configured with *compatible* settings

All nodes don't have to be the same, e.g. using cluster roles